interface/main/left_nav.php

```diff
@@ -816,7 +816,7 @@ function setTherapyGroup(group_id, group_name) {
      clearPatient();
 
      $(parent.Title.document.querySelector('#current_patient_block span.text')).hide();
-     setTitleContent('current_patient', '<span><?php echo xls('Therapy Group -');?> <a href=\'javascript:;\' onclick="parent.left_nav.loadCurrentGroupFromTitle(' + group_id +')">' + group_name + ' (' + group_id + ')<a></span>' );
+     setTitleContent('current_patient', '<span><?php echo xls('Therapy Group');?> - <a href=\'javascript:;\' onclick="parent.left_nav.loadCurrentGroupFromTitle(' + group_id +')">' + group_name + ' (' + group_id + ')<a></span>' );
```

Comment:
We don't include colons and hyphens in the translations constants. Otherwise would produce a lot of duplicative constants for translators.